frontend/www/js/omegaup/components/problem/FilterTags.test.ts

```diff
@@ -0,0 +1,35 @@
+import { mount } from '@vue/test-utils';
+import expect from 'expect';
+
+import { types } from '../../api_types';
+import T from '../../lang';
+
+import collection_Filter_Tags from './FilterTags.vue';
```

Comment:
Nit.
```suggestion
import collection_FilterTags from './FilterTags.vue';
```
También habrá que cambiarlo donde se utiliza.